Sources/ApolloWebSocket/WebSocketTransport.swift

```diff
@@ -74,12 +75,16 @@ public class WebSocketTransport {
               clientName: String = WebSocketTransport.defaultClientName,
               clientVersion: String = WebSocketTransport.defaultClientVersion,
               sendOperationIdentifiers: Bool = false,
+              reconnect: Bool = true,
               reconnectionInterval: TimeInterval = 0.5,
+              allowSendingDuplicates: Bool = true,
               connectingPayload: GraphQLMap? = [:],
               requestCreator: RequestCreator = ApolloRequestCreator()) {
     self.connectingPayload = connectingPayload
     self.sendOperationIdentifiers = sendOperationIdentifiers
+    self.reconnect.value = reconnect
```

Comment:
Since `reconnect` is initialized with `false`, it has to have been getting flipped somewhere if it's reconnecting. I don't see anywhere that's being changed here - is that on purpose?

Reply:
I think I didn't get the question. `reconnect` has default value to `true` in initilizer - WebSocketTransport.swift:78 

or I'm doing something wrong here?

Reply:
I was looking at WebSocketTransport.swift:28 where the `Atomic` var is initialized with a value of `false`. 

Because `reconnect.value` wasn't being flipped to `true` in the initializer previously, there's got to be someplace it's getting flipped elsewhere, which could make it ignore a `false` passed in with the initializer now that this has been explicitly added.

Reply:
[figured it out](https://github.com/apollographql/apollo-ios/pull/1004#discussion_r377881133) - further stuff can go through that comment